dist/getPkgInfo.js

```diff
@@ -28,18 +28,17 @@ var getPackageInfo = function getPackageInfo(packageId) {
       log('\t___');
       log();
       log('\t' + metadata.description);
-      log('\n\t' + title('GitHub Stars') + ': ' + numberWithCommas(github.starsCount));
-      log('\t' + title('Latest Commit was on') + ': ' + parseISOString(github.commits[0].to));
-      log('\t' + title('Open Issues') + ': ' + numberWithCommas(github.issues.openCount));
-      log('\t' + title('Total Issues') + ': ' + numberWithCommas(github.issues.count));
+      log('\n\t' + title('GitHub Stars') + ': ' + numberWithCommas(github && github.starsCount || ''));
+      log('\t' + title('Latest Commit was on') + ': ' + parseISOString(github && github.commits[0].to || ''));
+      log('\t' + title('Open Issues') + ': ' + (numberWithCommas(github && github.issues.openCount) || ''));
+      log('\t' + title('Total Issues') + ': ' + (numberWithCommas(github && github.issues.count) || ''));
       log();
-      log('\t' + title('GitHub') + ': ' + chalk.underline.cyan(metadata.links.repository));
-      log('\t' + title('Homepage') + ': ' + chalk.underline.cyan(metadata.links.homepage));
+      log('\t' + title('GitHub') + ': ' + chalk.underline.cyan(metadata.links.repository || ''));
+      log('\t' + title('Homepage') + ': ' + chalk.underline.cyan(metadata.links.homepage || ''));
       log('\t' + title('License') + ': ' + metadata.license);
-      log('\t' + title('Latest Release was on') + ': ' + parseISOString(metadata.releases[0].from));
+      log('\t' + title('Latest Release was on') + ': ' + parseISOString(metadata.releases[0].from || ''));
```

Comment:
Looks good to me but I think we should add a comprehensive message for the user.